UI/window-basic-settings-stream.cpp

```diff
@@ -290,6 +290,14 @@ void OBSBasicSettings::UpdateKeyLink()
 		streamKeyLink = "https://studio.trovo.live/mychannel/stream";
 	}
 
+	if (serviceName == "Dacast") {
+		ui->streamKeyLabel->setText(
+			QTStr("Basic.AutoConfig.StreamPage.EncoderKey"));
+	} else {
+		ui->streamKeyLabel->setText(
+			QTStr("Basic.AutoConfig.StreamPage.StreamKey"));
+	}
+
```

Comment:
You shouldn't need to add a new translation and a hard coded service check like this. Just go with stream key.

Reply:
We will revert this change.